### ISSUE TITLE
Screen titles not identified as headings [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#188]
The following screen titles are not announced as headings by VoiceOver.

- “Help stop the spread …” text on the Onboarding Introduction screen
- “Set Up Closeness Sensing …” text on the Onboarding ENS Intro screen
- “How it works” text on the How it works screens
- “Closeness Sensing and COVID Alerts are active” text on the Onboarding Confirm Success screen
- “NY COVID Data” text on the NY COVID Data screen
- “My Health Log” text on the My Health Log Intro screen
- “My Health Log” text on the My Health Log About screen
- “My Health Log” text on the My Health Log Symptoms List screen
- “Thank You” text on the My Health Log Results screens (both versions)
- “My COVID Alerts” text on the My COVID Alerts screen
- “If you test positive for COVID-19” text on the If you test positive for COVID-19 screen
- “How to Keep Others Safe” text on the How to Keep Others Safe screen
- “Share Your Close Contact Codes” text on the Share Your Close Contact Codes screen
- “Share Your Close Contact Codes” text on the Share Codes Confirmation screen
- “App Settings” text on the App Settings screen
- “My Health Log” text on the App Settings -> My Health Log screen
- “My COVID alerts” text on the App Settings -> My COVID Alerts screen
- “My Language” text on the App Settings -> My Language screen
- “Help Improve This App” text on the App Settings -> Help Improve This App screen
- “Data and Privacy Policy” text on the App Settings -> Data and Privacy Policy screen
- “Delete My Data” text on the App Settings -> Delete My Data screen

**Comment by willslone** [comment]
I retested and can confirm the issue has been resolved.
